src/ConfigProvider.php

```diff
@@ -69,15 +69,21 @@ public function getDependencies(): array
     public function getHalConfig(): array
     {
         return [
-            'resource-generator' => [
+            /**
+             * Whether or not to include empty collections within the _embedded section of the response,
+             *
+             * See: https://github.com/mezzio/mezzio-hal/pull/80
+             */
+            'embed-empty-collections' => false,
```

Comment:
At this point the information is hidden, the documentation needs to be expanded. For example here: https://docs.mezzio.dev/mezzio-hal/v2/links-and-resources/#resources

Reply:
Good point!